src/utils/searchAndSort.ts

```diff
@@ -37,7 +37,24 @@ export const splitTrigrams = (value: string): string[] => {
     }
   }
 
-  return trigrams;
+  trigrams.sort();
+  return new Set(trigrams);
+};
+
+export const trigramWordSimilarity = (a: string, b: string) => {
+  // Split the search value and result value into trigrams and compare their similarity
+  const aTrigrams = new Set(splitTrigrams(a));
+  const bTrigrams = new Set(splitTrigrams(b));
+
+  const matches: string[] = [];
+  bTrigrams.forEach((trigram) => {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/intersection

This may work the same? Not sure though

Reply:
I would love to use that, but unfortunately it isn't available in many browsers (no available at all in FF) - https://caniuse.com/?search=set.intersection.

Reply:
And I can't seem to find a babel polyfill for it.

Reply:
Sets are really that rarely used in Javascript I guess!

Reply:
Yea, it is quite sad. There is an [open proposal](https://tc39.es/proposal-set-methods/) for it so hopefully it will make it into the next release 🤞